docs/products/grafana/reference/plugins.rst

```diff
@@ -116,18 +116,21 @@ Worldmap Panel - `GitHub <https://github.com/grafana/worldmap-panel>`__
 
 Data source plugins
 -------------------
+Altinity plugin for ClickHouse® - `GitHub <https://github.com/Altinity/clickhouse-grafana>`__
```

Comment:
Could you fix the Vale warning on this one please?

Reply:
Done. Might need a rebase after the stuff that's been merged today?